iphone/Classes/TiUIPicker.m

```diff
@@ -232,6 +232,11 @@ - (void)setBackgroundColor_:(id)value
 
 - (void)setDateTimeColor_:(id)value
 {
+  // Guard date picker and iOS 14+ date picker style
+  if (![self isDatePicker] || [(UIDatePicker *)[self picker] preferredDatePickerStyle] != UIDatePickerStyleWheels) {
+    return;
+  }
```

Comment:
@hansemannn  One minor change required, otherwise it will crash for Xcode < 11.4
```suggestion
  if (![self isDatePicker]) {
    return;
  }
#if IS_SDK_IOS_13_4
  if ((UIDatePicker *)[self picker].preferredDatePickerStyle != UIDatePickerStyleWheels) {
    return;
  }
#endif
```